src/widgets/histogram/chart.js

```diff
@@ -467,11 +468,31 @@ module.exports = cdb.core.View.extend({
     this._onWindowResize();
   },
 
-  _setupScales: function () {
+  _getXScale: function () {
+    return d3.scale.linear().domain([0, 100]).range([0, this.chartWidth()]);
```

Comment:
Because those are the values of the brush extents: https://github.com/CartoDB/deep-insights.js/blob/master/src/widgets/histogram/chart.js#L547